content/en/docs/Writing policies/variables.md

```diff
@@ -405,11 +405,21 @@ regex_replace_all(regex string, src string|number, replace string|number) string
 regex_replace_all_literal(regex string, src string|number, replace string|number) string (converts all parameters to string)
 regex_match(string, string|number) bool
 label_match(object, object) bool (object arguments must be enclosed in backticks; ex. `{{request.object.spec.template.metadata.labels}}`)
+
 add(number, number) number
+add(quantity|number, quantity|number) quantity (returns quantity if any of the parameters is a quantity)
+add(duration|number, duration|number) duration (returns duration if any of the parameters is a duration)
 subtract(number, number) number
+subtract(quantity|number, quantity|number) quantity (returns quantity if any of the parameters is a quantity)
+subtract(duration|number, duration|number) duration (returns duration if any of the parameters is a duration)
 multiply(number, number) number
-divide(number, number) number (divisor must be non zero)
-modulo(number, number) number (divisor must be non-zero, arguments must be integers)
+multiply(quantity|number, quantity|number) quantity (returns quantity if any of the parameters is a quantity)
+multiply(duration|number, duration|number) duration (returns duration if any of the parameters is a duration)
+divide(quantity|number, quantity|number) quantity|number (returns quantity if exactly of the parameters is a quantity, else a number; the divisor must be non-zero)
```

Comment:
"returns quantity if all of the parameters are a quantity, else a number; ..."

Reply:
`multiply(number, quantity)` returns a `quantity`

Reply:
The comment was for divide.

Reply:
We're returning a quantity for the following cases:
```
divide(number, quantity)
divide(quantity, number)
divide(quantity, quantity)
```

Reply:
First one probably shouldn't return a quantity as that's technically an invalid calculation. But this is a functionality thing and not a docs thing.

Reply:
Yep, doesn't make sense mathematically but we decided to stick to this to keep things simple.

Reply:
So for that are we just assigning the same unit to the number as is present on the quantity? Ex., 10/5Mi is treated as 10Mi/5Mi?

Reply:
No, `10/5Mi` is interpreted as `10 Bytes / 5 Mi Bytes` = `2m`